source/audio/appsVolume.py

```diff
@@ -88,7 +89,7 @@ def _updateAppsVolumeImpl(
 	state: AppsVolumeAdjusterFlag,
 ):
 	global _activeCallback
-	if state == AppsVolumeAdjusterFlag.DISABLED:
+	if state in [AppsVolumeAdjusterFlag.DEFAULT, AppsVolumeAdjusterFlag.DISABLED]:
```

Comment:
If one day, the default state becomes "enabled" in the config spec, this statement will not work anymore. I guess that feature flags implement equality management so that comparing with default actually compares to its current value. It would be interesting to check here and to investigate why this equality management is not automatic here.

Reply:
Yeah I was also under impression that it should happen automatically. Hence this bug. But it's not likely appVolumeAdjuster will ever be enabled by default - there is a strong opposition to that from some folks here.

Reply:
I believe you want `FeatureFlag.calculated()` here.
```suggestion
	if state.calculated() == AppsVolumeAdjusterFlag.DISABLED:
```